packages/server/src/index.ts

```diff
@@ -89,7 +89,7 @@ export const start = async (
         );
       })
       .on("error", e => {
-        if (portNumber - startingPort > portTriesLimit) {
+        if (portNumber - port > portTriesLimit) {
```

Comment:
This seems unrelated.

Reply:
You are correct, I'll put this in another PR

Reply:
Maybe squash before merging to avoid complication.